internal/tree.go

```diff
@@ -12,7 +12,23 @@ import (
 	"github.com/xlab/treeprint"
 )
 
-func renderTreeView(tasks []task) error {
+var (
+	Name           string
+	Version        string
+	Format         string
+	Stats          bool
+	Mounts         bool
+	OutputFilePath string
+	PID            uint32
+	PsTree         bool
+	PsTreeCmd      bool
+	PsTreeEnv      bool
+	Files          bool
+	Sockets        bool
+	ShowAll        bool
```

Comment:
These are common to both the tree and JSON views, so it would make more sense to have it in a separate file (possibly `options.go`?)

Reply:
Should I do the same for the cmd package where the variables are used by all three commands?